### ISSUE TITLE
HTML and CSS procedure article

### ISSUE THREAD

**Issue by MinervaLong** [chingu-voyages/v11-toucans-team-13#28]
https://jdauriemma.com/programming/methodical-html-css-development

I use this article for my pre-work an it helps me a lot, we could use this procedure as a order step guide

**Comment by dan9d9** [comment]
Yeah, I used this too! I posted it in the Chingu resource channel, haha. I agree, I think we should use this procedure.

**Comment by MinervaLong** [comment]
¿it was you? Thank so much! xD
